### ISSUE TITLE
The image space setting is broken. PROPFIND and GET methods give 404 

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/ocis#3663]
ownCloud Web UI 5.5.0
ocis via docker v.1.20

Steps: 

- admin creates space
- admin opens context menu and click "edit image"
- admin uploads image **jpg:**

Actual result:
request1: uploading file -PUT works correctly `https://host.docker.internal:9200/remote.php/dav/spaces/1284d238-aa92-42ce-bdc4-0b0000009157%243e85a541-bb87-4a22-a83f-749886e3b55d/.space/Lake-Constazdxcnce.jpg`

request2: setting file as space image - PATCH works correctly  `https://host.docker.internal:9200/graph/v1.0/drives/1284d238-aa92-42ce-bdc4-0b0000009157%243e85a541-bb87-4a22-a83f-749886e3b55d`

request3: getting image image PROPFIND works correctly `https://host.docker.internal:9200/remote.php/dav/spaces/1284d238-aa92-42ce-bdc4-0b0000009157%243e85a541-bb87-4a22-a83f-749886e3b55d/.space/Lake-Constazdxcnce.jpg`

Actual result:
request 4- 404 Not found: GET`https://host.docker.internal:9200/remote.php/dav/spaces/1284d238-aa92-42ce-bdc4-0b0000009157%243e85a541-bb87-4a22-a83f-749886e3b55d/.space/Lake-Constazdxcnce.jpg?scalingup=0&preview=1&a=1&c=e4e33738f68aab8d755bb5027c108890&x=1200&y=1200`

![Screenshot 2022-05-03 at 17 40 51](https://user-images.githubusercontent.com/84779829/166486977-1ed43488-c8a0-4e5e-8663-ece222ceef47.png)

### or if you try to upload png image
- admin uploads image **png:**
request 3 PROPFIND `https://host.docker.internal:9200/remote.php/dav/spaces/1284d238-aa92-42ce-bdc4-0b0000009157%240d536bf9-b562-4aa2-80a9-08053c80ce04/.space/Screenshot%25202022-05-03%2520at%252009.27.10.png` - get 404 error
`<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns"><s:exception>Sabre\DAV\Exception\NotFound</s:exception><s:message>Resource not found</s:message></d:error>`

![image](https://user-images.githubusercontent.com/84779829/166491659-78b0d5fd-49b6-4a4e-b2c8-48ed10d6e352.png)

**Comment by C0rby** [comment]
This has been fixed and works again in ocis master.

**Comment by C0rby** [comment]
Yes, you're right. It seems that the file name is double URL encoded. Let me check.

**Comment by C0rby** [comment]
@kulmann, it seems like web encodes the webdav URL again before using it to download the space image.
Last week or so we changed the webdav URL to already come URL encoded so this issue might be fixed by removing the second encoding in web. Could you plan this in the web team?
